### DISCUSSION TITLE
PCAPs Never Complete Always Pending.

### DISCUSSION THREAD

**Discussion by Mfisher925** [Security-Onion-Solutions/securityonion#8178] [Unsupported Versions]
Hello,

I have a distributed deployment with 1 Manager, 2 Search, and Many forward nodes.  Sec. Onion version 2.3.130

I am having trouble pulling PCAPs from some nodes and not others.  The PCAPs stay "Pending" and never complete.

I reviewed the logs at /opt/so/log/sensoroni/sensoroni.log and there are no errors.  I see the PCAP job start but nothing happens after that.

I don't see any resource strain on the Host/VM guests.  Disk read/write, processor use are in the green.

The specs for the Forward node are below, the typical EPS for these nodes are higher than the others that are working, around 60-100 EPS.  However there is a node with higher EPS that processes PCAPs just fine.

CPU	8 vCPUs
Memory	16 GB
Hard disk 1	400 GB
USB controller	USB 2.0
Network adapter 1	VM Network (Connected) GB Ethernet
Network adapter 2	Sniff_Port_Group (Connected) GB Ethernet

Resource Consumption
Consumed host CPU	1.7 GHz
Consumed host memory	10.95 GB
Active guest memory	4.16 GB

**Comment by weslambert** [comment]
Has pulling PCAP from the affected node(s) ever worked, or was this issue recently introduced?
